quickstart for data ingestion tools and use cases (#101)
Teradata/quickstarts: modules/ROOT/pages/recommended-data-ingestion-tools-in-different-use-cases-for-teradata-vantage.adoc

* *Teradata Parallel Transporter (TPT)*: It is recommended option to load data from local files when high throughput is needed using scalability and parallelism.  

* *BTEQ*: BTEQ has full scripting capabilities and can read files. It is a good option if it is already a dominant ingestion tool with a customer.

=== Move the data from different systems for unified query processing
==== Available Tools: link:https://docs.teradata.com/r/Teradata-QueryGridTM-Installation-and-User-Guide/October-2020/Teradata-QueryGrid-Overview[Teradata QueryGrid,window="_blank"]

*Teradata QueryGrid*: It is a recommended option to move limited quantities of data between different systems/platforms and that includes movement within Vantage instances, Apache Spark, Oracle, Presto etc when what needs to be synced is described by complex conditions that can be expressed in SQL. 

=== Ingesting data from SaaS applications (Third Party Tools)
==== Available Tools: link:https://airbyte.com/[Airbyte,window="_blank"]

> *Airbyte*: Airbyte is an ELT tool that has more than 350 connectors and is Open Source. It is a preferred choice to use if you want to perform lightweight ingestions from SaaS applications into Teradata Vantage.

Reviewer: ```suggestion
*Airbyte*: Airbyte is an ELT tool that has more than 350 connectors and is Open Source. It's a favored option for conducting lightweight ingestions from SaaS applications into Teradata Vantage.
```

== Summary

> In this article we covered different use cases of data ingestion, list of available tools and the recommended tools to be used in different use cases. 

Reviewer: ```suggestion
In this article, we explored various data ingestion use cases, provided a list of available tools for each use case, and identified the recommended option for different scenarios.
```
